Avatar.css

```diff
@@ -8,13 +8,30 @@
   display: flex;
   flex-wrap: wrap;
   flex-direction: row;
-  justify-content: center;
+  /* justify-content: center; */
 }
 
-.avatar{
+.avatar {
+  border-radius: 80px;
+  /* margin: 100px; */
+}
 
-    height: 100px;
```

Comment:
use rem instead of px. It will help you later in making your website responsive.